ff/src/lib.rs

```diff
@@ -152,6 +155,7 @@ pub trait PrimeFieldRepr:
     fn shl(&mut self, amt: u32);
 
     /// Writes this `PrimeFieldRepr` as a big endian integer.
+    #[cfg(feature = "std")]
     fn write_be<W: Write>(&self, mut writer: W) -> io::Result<()> {
```

Comment:
This API will be replaced with `no_std`-compatible `to_bytes` (matching the `bls12_381` and `jubjub` APIs) when we remove `PrimeFieldRepr`.